panther_manager/plugins/action/call_trigger_service_node.cpp

```diff
@@ -3,26 +3,19 @@
 namespace panther_manager
 {
 
-CallTriggerService::CallTriggerService(const std::string & name, const BT::NodeConfig & conf)
-: RosServiceNode(nh_, name, conf)
-{
-  nh_ = config().blackboard->get<std::shared_ptr<ros::NodeHandle>>("nh");
-  getInput("service_name", srv_name_);
-}
-
 void CallTriggerService::update_request(RequestType & request) {}
```

Comment:
```suggestion
```
I don't think there is a point of defining it in the cpp file, since this function does exactly nothing